action/src/main.ts

```diff
@@ -1,26 +1,52 @@
-import * as core from '@actions/core'
-import { wait } from './wait'
+import * as core from '@actions/core';
```

Comment:
not gonna cry about relative imports lol

Reply:
yeah lolol i didn't want to deal with configuring it. could be a nice QOL update as a ff